### DISCUSSION TITLE
About PK-Analysis in Simulation

### DISCUSSION THREAD

**Discussion by Anhng1234** [Open-Systems-Pharmacology/Forum#1221] [Models]
Could you guys please help me with why I have this warning? I can not find where the problem is? 
![PK sim problem](https://user-images.githubusercontent.com/100749525/171543302-f44e6a8b-04b6-4fc4-9580-8c3b1c0b447f.png)

Thank you so so much for your help, I am looking forward to hearing from you. 
Have a nice day!

**Comment by StephanSchaller** [comment]
You have to simulate for a longer time. The program basically tells you that where the simulation stops, not all is absorbed, yet...
